source/docs/software/frc-glossary.rst

```diff
@@ -11,21 +11,51 @@ FRC Glossary
    auto
       The first phase of each match is called Autonomous (auto) and consists of the robot's running pre-programmed instructions.
 
+   call stack
+      A specially-organized region of memory which helps the program keep track of what function it is in, and store the values of some variables. See `call stack <https://en.wikipedia.org/wiki/Call_stack>`__ on wikipedia for more info.
+
+   Classical Mechanics
+      The branch of physics which studies and describes the motion of relatively large, relatively slow objects. See `Classical Mechanics <https://en.wikipedia.org/wiki/Classical_mechanics>`__ on wikipedia for more info.
+
    COTS
       Commercial off the shelf, a standard (i.e. not custom order) part commonly available from a vendor to all teams for purchase.
 
+   compositions
```

Comment:
I'd leave this as singular ("composition") and describe it as "A term for building (or "composing") software entities out of other, smaller ("component") software entities."